src/Cache_Command.php

```diff
@@ -406,4 +409,169 @@ public function flush_group( $args, $assoc_args ) {
 		}
 		WP_CLI::success( "Cache group '$group' was flushed." );
 	}
+
+	/**
+	 * Get a nested value from the cache.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Cache key.
+	 *
+	 * <key-path>...
+	 * : The name(s) of the keys within the value to locate the value to pluck.
+	 *
+	 * [--group=<group>]
+	 * : Method for grouping data within the cache which allows the same key to be used across groups.
+	 * ---
+	 * default: default
+	 * ---
+	 *
+	 * [--format=<format>]
+	 * : The output format of the value.
+	 * ---
+	 * default: plaintext
+	 * options:
+	 *   - plaintext
+	 *   - json
+	 *   - yaml
+	 * ---
+	 */
+	public function pluck( $args, $assoc_args ) {
+		list( $key ) = $args;
+		$group       = Utils\get_flag_value( $assoc_args, 'group' );
+
+		$value = wp_cache_get( $key, $group );
+
+		if ( false === $value ) {
+			WP_CLI::warning( 'No object found for the key "' . $key . '" in group "' . $group . '"' );
```

Comment:
```suggestion
			WP_CLI::warning( "No object found for the key '$key' in group '$group'" );
```

Since we are using double quotes further down as well, makes it more consistent.